example_dags/README.md

```diff
@@ -0,0 +1,93 @@
+# Running Example DAGs
+
+This directory contains several example Airflow DAGs to demonstrate how to use the
```

Comment:
Let's have two sub folder for the two different examples, so it is easier to read?

Reply:
Mention what are the two examples, i.e. what they solve.

Reply:
I think having subfolders might not be necessary, since the "how to run it" is the same. But I agree that we should add a brief description of all the examples though.